<launch>

	<arg name="rviz" default="true" />

	<node pkg="ligo" type="ligo_mapping" name="laserMapping" output="screen">
	<rosparam command="load" file="$(find ligo)/config/avia.yaml" />
	<param name="prop_at_freq_of_imu" type="bool" value="1"/>
	<param name="check_satu" type="bool" value="1"/>
	<param name="init_map_size" type="int" value="10"/>
	<param name="point_filter_num" type="int" value="4"/> 
	<param name="space_down_sample" type="bool" value="1" />  
	<param name="filter_size_surf" type="double" value="0.3" /> 
	<param name="filter_size_map" type="double" value="0.3" /> 
	<param name="ivox_nearby_type" type="int" value="6" /> 
	<param name="runtime_pos_log_enable" type="bool" value="0" />
	</node>
	<group if="$(arg rviz)">
	<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find ligo)/rviz_cfg/loam_livox.rviz" />
	</group>

	launch-prefix="valgrind" 

</launch>